README.md

```diff
@@ -34,6 +34,7 @@ for the following modules.
 * akka-discovery-aws-api
 * akka-discovery-aws-api-async
 * akka-discovery-consul
+* loglevels-logback
```

Comment:
Not sure if we want to mark it as supported right away rather?

Reply:
yeah, not much to incubate